neo/rawio/neuralynxrawio.py

```diff
@@ -89,7 +89,8 @@ def _parse_header(self):
             if ext not in self.extensions:
                 continue
 
-            if os.path.getsize(filename)<=HEADER_SIZE:
+            if (os.path.getsize(filename)<=HEADER_SIZE) and \
+               (ext in ['nse', 'ncs', 'ntt']):
                 self._empty_files.append(filename)
                 continue
```

Comment:
I would put only 'ncs' because emty spike channel could be also a relevant object. No ?

Reply:
On the one hand, true, when we have a good channel with empty spikes, this is something informative, on the other hand, if we have an empty channel and an empty spike channel, this must be skipped.

Reply:
So, should I go with just NCSs?

Reply:
Lets wait for @JuliaSprenger opinion.

Reply:
So, should I go with just NCSs?

Reply:
I agree with @samuelgarcia, an empty nse or ntt file is still important information we should provide. In principle this would even be true for ncs files, but there we just don't have enough info in the file header to be able to generate an empty AnalogSignal or similar.

Reply:
@JuliaSprenger Done!